packages/@aws-cdk/aws-cloudfront-origins/lib/s3-origin.ts

```diff
@@ -79,7 +80,16 @@ class S3BucketOrigin extends cloudfront.OriginBase {
       this.originAccessIdentity = new cloudfront.OriginAccessIdentity(oaiScope, oaiId, {
         comment: `Identity for ${options.originId}`,
       });
-      this.bucket.grantRead(this.originAccessIdentity);
```

Comment:
Isn't there a grant() API on bucket that takes a list of actions?

Reply:
There's a `private` one:
https://github.com/aws/aws-cdk/blob/master/packages/@aws-cdk/aws-s3/lib/bucket.ts#L716

I briefly toyed with the idea of adding a `grantReadObject` method, but felt it was a bit niche.

Reply:
It's a common pattern for constructs to expose a `grant(grantee, ...actions)` API for such niche use cases.